stylesheets/partials/features/_events.scss

```diff
@@ -460,4 +460,16 @@ a, button, input, label {
 			background-position: top;
 		}
 	}
+
+	.events-list {
+		.cards {
+			.card {
+				.event-name {
```

Comment:
does this need to be this specific?

can it just be

`.event-name` or `.event-list .event-name`?

Reply:
Yes -- I only want this to happen inside a card.